README.md

```diff
@@ -90,6 +97,14 @@ You do not have sufficient access to the device. You probably want to add the a
 
 <!-- markdownlint-enable MD013 -->
 
+## Interference from CUPS
+
+Sometimes, the CUPS printing system will attempt to communicate with the label
+maker, causing interference with Labelle.  If you don't otherwise have a USB
```

Comment:
Not necessary, but do you have any ideas for if someone does have a USB printer? For example, is it possible to make this rule only apply to a specific USB device?

Reply:
I made an effort at that, but couldn't figure out how to block the system udev rules that way.  There may be a way but I couldn't find it.